op-node/rollup/driver/state.go

```diff
@@ -200,11 +199,12 @@ func (s *Driver) eventLoop() {
 		sequencerTimer.Reset(delay)
 	}
 
-	// Create a ticker to check if there is a gap in the engine queue every 15 seconds
-	// If there is, we send requests to the backup RPC to retrieve the missing payloads
-	// and add them to the unsafe queue.
-	altSyncTicker := time.NewTicker(15 * time.Second)
+	// Create a ticker to check if there is a gap in the engine queue. Whenever
+	// there is, we send requests to sync source to retrieve the missing payloads.
+	syncCheckInterval := time.Duration(s.config.BlockTime) * time.Second * 2
```

Comment:
Why are we casting blocktime to a duration. Blocktime is in seconds, duration is in NS (https://pkg.go.dev/time#Duration)

Reply:
We're multiplying it to `time.Second`. We need to cast to duration to get it in the right type for the multiplication. I can also cast `time.Second` to uint64 first and then cast the result of multiplication, if you prefer.

Reply:
Sorry, I had a brain fart looking at the cast. That's correct.